.swiftlint.yml

```diff
@@ -21,7 +21,7 @@ excluded: # paths to ignore during linting
 - Carthage
 - SampleApps/*/Pods
 - Pods
-
+- .build
```

Comment:
Is this supposed to be "build" (without the '.')?

Reply:
`.build` is the default build folder for SPM, so it may include dependencies which we don't want to include when running the lint command.